plugins/module_utils/azure_rm_common_ext.py

```diff
@@ -281,17 +306,13 @@ def update_single_managed_identity(self, curr_identity, new_identity, patch_supp
             new_identity_converted['user_assigned_identities'] = {
                 'id': [user_assigned_identity]
             }
-        return self.update_managed_identity(curr_identity=curr_identity,
-                                            new_identity=new_identity_converted,
+        return self.update_managed_identity(new_identity=new_identity_converted,
+                                            curr_identity=curr_identity,
                                             allow_identities_append=False, patch_support=patch_support)
 
-    def update_managed_identity(self, curr_identity=None, new_identity=None,
+    def update_managed_identity(self, new_identity, curr_identity=None,
```

Comment:
The arguments passed in the module call are all keyword arguments, and azure_rm_sqlserver.py 375 lines also need to be updated
```suggestion
    def update_managed_identity(self, new_identity={}, curr_identity=None,
```